package.json

```
@@ -81,9 +82,11 @@
     "start-heroku": "tt-serve-static --port $PORT --site './build' ",
     "start-prod": "tt-serve --env.disableHotReloading -p",
     "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run jest && npm run wdio",
+    "test": "npm run jest && npm run wdio && npm run wdio-fusion",
     "update-package-readme": "node scripts/root-readme-updater/index.js",
-    "wdio": "wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js"
+    "wdio-default": "wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
```

For the form factors argument to work, does this need to be tt-wdio instead of just wdio? @emilyrohrbough 
```suggestion
    "wdio-default": "tt-wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
```

---

let's see https://github.com/cerner/terra-clinical/pull/560/commits/586737f2e5a2003dd79b3f137ac0bb80f20be52d